### PR TITLE
Update troubleshooting.rst

### PR DESCRIPTION
**Pull request by joewaitye**
Added an entry about issues when configuring the Giphy integration, and its solution

### CODE REVIEW

File: source/install/troubleshooting.rst [hunk 1]
```diff
@@ -100,6 +100,11 @@ Mattermost can't connect to LDAP/AD server
 
 LDAP and Active Directory troubleshooting can be found on `this page. <https://docs.mattermost.com/deployment/sso-ldap.html#troubleshooting-faq>`__
 
+Hitting into error "Command with a trigger of failed" when configuring Giphy integration
```
Comment (amyblais) [inline source/install/troubleshooting.rst]:
```suggestion
Hitting an error "Command with a trigger of failed" when configuring Giphy integration
```

File: source/install/troubleshooting.rst [hunk 2]
```diff
@@ -100,6 +100,11 @@ Mattermost can't connect to LDAP/AD server
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When trying to configure the Giphy integration in Mattermost, you may hit into the error of "Command with a trigger of <keyword> failed". To solve this, you would need to edit your ``config.json`` and configure ``AllowedUntrustedInternalConnectionsneeds`` to contain the hostname of the webhook.
```
Comment (amyblais) [inline source/install/troubleshooting.rst]:
```suggestion
When trying to configure the Giphy integration in Mattermost, you may hit the error "Command with a trigger of <keyword> failed". To solve this, you need to edit your ``config.json`` and configure ``AllowedUntrustedInternalConnections`` to contain the hostname of the webhook.
```

**Comment by levb** [comment]
I am not sure what the context here is, was there a ticket?

**Comment by joewaitye** [comment]
Not related to any specific ticket, but from personal experience trying to integrate giphy with Mattermost. I have signed the CLA, but it is still telling me that I need to sign it to merge the changes, any ideas?

**Comment by amyblais** [comment]
@joewaitye PRs should be reviewed and approved before merging. Not sure why it gives the error about the CLA, but I can help merging this if you can take a look at the readability suggestions that I noted above ^

**Comment by joewaitye** [comment]
Hi @amyblais so sorry for the late reply. I am okay with the changes you suggested, and have committed them. I cannot seem to merge this pull request as it states I do not have write access, so going to need your help on this one.

**Pull request merged**